RFS/src/main/java/com/rfs/version_es_6_8/ElasticsearchConstants_ES_6_8.java

```diff
@@ -1,11 +1,16 @@
 package com.rfs.version_es_6_8;
 
+import com.fasterxml.jackson.core.JsonFactory;
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.core.JsonParser;
 import com.fasterxml.jackson.dataformat.smile.SmileFactory;
 import com.fasterxml.jackson.dataformat.smile.SmileGenerator;
 
 public class ElasticsearchConstants_ES_6_8 {
+    private ElasticsearchConstants_ES_6_8() {
```

Comment:
By adding an exception this requires a test to cover it, better off to use @UtilityClass from lombok

Reply:
Great suggestion, thanks!